dev/ColorPicker/readme.md

```diff
@@ -0,0 +1,141 @@
+* `HSVComboBoxItem` : [ComboBoxItem] The item specifying 'HSV' color representation within the `ColorRepresentationComboBox`.
+
+* `RedTextBox` : [TextBox] The red channel input text box the user can modify.
+* `GreenTextBox` : [TextBox] The green channel input text box the user can modify.
+* `BlueTextBox` : [TextBox] The blue channel input text box the user can modify.
+* `HueTextBox` : [TextBox] The hue channel input text box the user can modify.
+* `SaturationTextBox` : [TextBox] The saturation channel input text box the user can modify.
+* `ValueTextBox` : [TextBox] The value channel input text box the user can modify.
+* `AlphaTextBox` : [TextBox] The alpha channel input text box the user can modify (shared for both RGB/HSV color representation). This value is a percentage from 0%..100% and the '%' symbol will be automatically added and cannot be removed.
+* `HexTextBox` : [TextBox] The hexadecimal color input text box the user can modify. This is always in RGB color representation in either #RGB or #ARGB format. The '#' symbol will be automatically added and cannot be removed.
+
+* `RedLabel` : [TextBlock] A localized label to identify the red channel input text box. 
+* `GreenLabel` : [TextBlock] A localized label to identify the green channel input text box.
+* `BlueLabel` : [TextBlock] A localized label to identify the blue channel input text box.
+* `HueLabel` : [TextBlock] A localized label to identify the hue channel input text box.
+* `SaturationLabel` : [TextBlock] A localized label to identify the saturation channel input text box.
+* `ValueLabel` : [TextBlock] A localized label to identify the value channel input text box.
+* `AlphaLabel` : [TextBlock] A localized label to identify the alpha channel input text box (shared for both RGB/HSV).
+
+## ColorPickerSlider
+
+A custom slider implementation used for the sliders in the `ColorPicker` control template. These sliders are unique in that they have a gradient background rendered to correspond with a defined HSV color channel (important notes on background below). Sliders provide a one-dimensional method of modifying a color channel. They are used for the third color channel not on the `ColorSpectrum` or the alpha channel.
+
+Visually to the user, these sliders have a background rendered based on the assigned HSV color channel or alpha. For example, the background could be a gradient of the selected color from minimum saturation to maximum saturation if the slider represents the saturation channel. However, the slider itself does not draw this background gradient. The gradient is managed by the parent `ColorPicker` and shown underneath the slider as the fill of a background rectangle. For the alpha channel slider there is a further checkered background that is rendered separately and then blended using opacity. `ColorPickerSlider` and `ColorPicker` are intrinsically linked together and `ColorPickerSlider` is not useful as a stand-alone control.
```

Comment:
You say ColorPickerSlider is not a useful as a stand-alone control, yet in the last section you argue that it should be made an independent control. Also is it relevant to mention what is useful as a stand-alone control? Isn't that something everyone has to decide for themselfes and not something the documentation decides?

Reply:
Linking two controls together like this isn't very good architecture which is the fundamental concern. Additionally, a lot can be simplified by cleanly separating these controls. That simplification is mentioned.

Reply:
> You say ColorPickerSlider is not a useful as a stand-alone control

Is as it's coded now.

> yet in the last section you argue that it should be made an independent control.

Last section is future areas of improvement.

These are unrelated topics.

Reply:
It's still sending mixed messages here. But that's not my main concern, in my opinion documentation shouldn't say "part x of our control is not useful for you". In my opinion documentation should be neutral, not opinionated. But if you think it's fine to include that, then do that, just saying my opinion here.

Reply:
It's important that developers don't think they can just use the ColorPickerSlider and expect the background gradient to be rendered. It is important they understand it is done by the ColorPicker itself. This is not an opinion.

Reply:
Then say that the background does not render not on it's own instead of "not helpful on it's own". The fact that it doesn't render the background is a fact yes, but if it's useful for me or not is still up to me :)

Reply:
"Is not intended as a standalone control". Would dial down the wording but the intent stays the same. You are interpreting this very strongly and I don't think others would necessarily read it that way.

Reply:
Could be that I am interpreting "is not useful" too strongly here, but might not be the only one concerned by that.

Reply:
It might be better to say that the ColorPickerSlider is not intended to be used outside of the context of a ColorPicker than to say that it's "not useful".  It was created to meet the needs of the ColorPicker control; I can't think of any way it would be useful as a standalone control to be used by itself, which is why it's under the Primitives namespace.  It's effectively just a custom part of ColorPicker.

Reply:
I find the spectrum is quite useful by itself in custom color picker implementations even though it's considered a primitive. The slider though, not so much.